File: Learnum_ERP UI/src/app/tds/component/hrd/trainer/add-trainer/trainerdetails.model.ts
export class TrainerDetailsModel {
    TrainerId: number;
    CourseId: number;
    BranchId: number;
    BatchId: number;
    SubjectId: number;
    SubjectName: string;
    BranchName: string;
    BatchName: string;
    CourseName: string;
    TrainerName: string;
    isActive: boolean;
    AddedBy: number | null;
    AddedDate: Date | null;
    UpdatedBy: number | null;
    UpdatedDate: Date | null;
}

File: Learnum.ERP.Shared/Entities/Models/TrainerDetailsModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Http;

namespace Learnum.ERP.Shared.Entities.Models
{
    public class TrainerDetailsModel : BaseModel
    {

        public long TrainerId { get; set; }
        public long CourseId { get; set; }
        public long SubjectId { get; set; }
        public long BranchId { get; set; }
        public long BatchId { get; set; }
        public string TrainerName { get; set; }
        public bool IsActive { get; set; }

    }
       
       
    }

